src/Hng.Application/Features/Organisations/Dtos/OrganizationUserDto.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Hng.Domain.Entities;
+using System.Text.Json.Serialization;
+
+namespace Hng.Application.Features.Organisations.Dtos
+{
+    public class OrganizationUserDto
+    {
+        public ICollection<User> Users { get; set; }
```

Comment:
Dont use the entity type User directly, It'll expose non user friendly details, create a custom dto for it that resembles the User Entity, add to add to automapper profile

also include jsonpropertyname